### DISCUSSION TITLE
metalLB IPs and DNS

### DISCUSSION THREAD

**Discussion by ColdCanuck** [techno-tim/k3s-ansible#27] [Questions] [answered]
Completely new to k8s, so pardon the newbie question.

Is there a canonical way to get the IP assigned for a service by metalLB into my DNS. I understand that as services come and go,  the assigned IP for that particular service is assigned from the "pool", but the specific assignment might be different one time to the next. I can see how I can scrape for the information with say `kubectl describe service serviceX` and I could write a script to do this periodically and update my DNS server, but is that the standard way ?

**Comment by timothystewart6** [comment, marked as answer]
Funny you ask, I just wrote this to do the same thing


`kubectl get services nginx -o jsonpath="{.status.loadBalancer.ingress[0].ip}"`

Replace it with your service name
